### ISSUE TITLE
Error parsing ECDSA public key

### ISSUE THREAD

**Issue by pathornteng** [hashgraph/hedera-sdk-js#1358]
### Description

Currently, Javascript SDK is throwing an error when it tries to parse ECDSA public. 

```
const newAccountPrivateKey = PrivateKey.generateECDSA();
const newAccountPublicKey = newAccountPrivateKey.publicKey;
const publicKey2 = PublicKey.fromString(newAccountPublicKey.toStringDer());
```

This problem does not happen with ED25519.

### Steps to reproduce

Run the following code
```
const newAccountPrivateKey = PrivateKey.generateECDSA();
const newAccountPublicKey = newAccountPrivateKey.publicKey;
const publicKey2 = PublicKey.fromString(newAccountPublicKey.toStringDer());
```
You will get 
```
BadKeyError: invalid public key length: 47 bytes
    at Function.fromBytes (/Users/pathorn/Work/hedera/hedera-basic/node_modules/@hashgraph/cryptography/lib/PublicKey.cjs:74:11)
    at Function.fromString (/Users/pathorn/Work/hedera/hedera-basic/node_modules/@hashgraph/cryptography/lib/PublicKey.cjs:106:22)
    at Function.fromString (/Users/pathorn/Work/hedera/hedera-basic/node_modules/@hashgraph/sdk/lib/PublicKey.cjs:103:49)
    at main (/Users/pathorn/Work/hedera/hedera-basic/create-account.js:43:32)
    at Object.<anonymous> (/Users/pathorn/Work/hedera/hedera-basic/create-account.js:66:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
```

### Hedera network

mainnet, testnet, previewnet

### Version

lastest

### Operating system

macOS

**Comment by pathornteng** [comment]
I suspect that we have the same problem with Java and Golang SDKs

**Comment by petreze** [comment]
@pathornteng 
Tried to run the provided code example but the issue did not appear. I've tried also with the `async` version and it was again generated properly. Do you have a different setup or something that you might have missed on this one?

**Comment by pathornteng** [comment]
I am using v2.19.0, the latest version, and I get the error every time. The following is the complete script on MacOS.

```
const { Client, PrivateKey, PublicKey } = require("@hashgraph/sdk");

async function main() {
  const newAccountPrivateKey = PrivateKey.generateECDSA();
  const newAccountPublicKey = newAccountPrivateKey.publicKey;
  const publicKey2 = PublicKey.fromString(newAccountPublicKey.toStringDer());
}
main();
```

**Comment by petreze** [comment]
@pathornteng 
Hello, try using `newAccountPublicKey.toStringRaw()` instead of `newAccountPublicKey.toStringDer()`

**Comment by pathornteng** [comment]
Hi @petreze, yes  `newAccountPublicKey.toStringRaw()` works but `newAccountPublicKey.toStringDer()` should also work since it works for ED25519.

**Comment by ochikov** [comment]
@pathornteng We are going to fix it in the new release. Thank you for the opened issue.

**Comment by ochikov** [comment]
Fixed with #1366
